wagtail/admin/views/chooser.py

```diff
@@ -93,8 +93,8 @@ def browse(request, parent_page_id=None):
 
     parent_page = parent_page.specific
 
-    # Get children of parent page
-    pages = parent_page.get_children().specific()
+    # Get children of parent page (without streamfields)
+    pages = parent_page.get_children().defer_streamfields().specific()
```

Comment:
Question: is the reason you're using this rather than `specific(defer=True)` in case something like the `admin_display_title` method on the specific model actually references one of the deferred fields, causing a query per row? Or is there another consideration here?

Reply:
You are totally right, yes. This is so that we can still effectively support those kinds of overrides for the time being. Hopefully we can avoid `specific()` completely at a later time, but for now, this should alleviate things somewhat

Reply:
Hmm, looks like my previous comment didn't save! But anyway - in that case, that sounds like a good measure. I agree with Karl that it would be nice to remove specific usage entirely from the admin eventually, but this sounds like a really good idea in the short term that doesn't require the same kind of deprecation path.